### PR TITLE
Update LDAP group documents

### PR DESCRIPTION
**Pull request by stonezdj**
Signed-off-by: stonezdj <stonezdj@gmail.com>

### CODE REVIEW

File: docs/manage_role_by_ldap_group.md [hunk 1]
```diff
@@ -17,18 +17,23 @@ This guide provides instructions to manage roles by LDAP/AD group. You can impor
 
 Besides **[basic LDAP configure parameters](https://github.com/vmware/harbor/blob/master/docs/installation_guide.md#optional-parameters)** , LDAP group related configure parameters should be configured, they can be configured before or after installation
 
-  1. Configure parameters in harbor.cfg before installation
+  1. Configure LDAP parameters in with command line, please refer **[Config Harbor user settings by command line](configure_user_settings.md)**
```
Comment (ywk253100) [inline docs/manage_role_by_ldap_group.md]: It should be `Configure LDAP parameters with command line`, no `in`, and it should be `refer to`.

File: docs/manage_role_by_ldap_group.md [hunk 2]
```diff
@@ -17,18 +17,23 @@ This guide provides instructions to manage roles by LDAP/AD group. You can impor
...
    * ldap_group_gid    -- The attribute used to name an LDAP/AD group, for example: cn 
    * ldap_group_scope  -- The scope to search for LDAP/AD groups. 0-LDAP_SCOPE_BASE, 1-LDAP_SCOPE_ONELEVEL, 2-LDAP_SCOPE_SUBTREE 
 
-  2. Or Change configure parameter in web console after installation. Go to "Administration" -> "Configuration" -> "Authentication" and change following settings.
-   - LDAP Group Base DN -- ldap_group_basedn in harbor.cfg
-   - LDAP Group Filter  -- ldap_group_filter in harbor.cfg
-   - LDAP Group GID     -- ldap_group_gid in harbor.cfg
-   - LDAP Group Scope   -- ldap_group_scope in harbor.cfg
+  2. Or change configure parameter in web console after installation. Go to "Administration" -> "Configuration" -> "Authentication" and change following settings.
+   - LDAP Group Base DN -- ldap_group_basedn in the Harbor user settings
```
Comment (ywk253100) [inline docs/manage_role_by_ldap_group.md]: The `Harbor user settings` isn't clearly clarify what it is

File: docs/manage_role_by_ldap_group.md (hunk 1 above)
Comment (reasonerjt) [reply docs/manage_role_by_ldap_group.md]: how about rephrase to `Configure LDAP parameters via API`

File: docs/manage_role_by_ldap_group.md (hunk 2 above)
Comment (stonezdj) [reply docs/manage_role_by_ldap_group.md]: All user setting is documented in https://github.com/goharbor/harbor/blob/master/docs/configure_user_settings.md, already linked to this doc

File: docs/manage_role_by_ldap_group.md (hunk 1 above)
Comment (stonezdj) [reply docs/manage_role_by_ldap_group.md]: OK

**Pull request merged**